README.rst

```diff
@@ -1,225 +1,147 @@
-   replace ``clickhouse`` with ``{{CLICKHOUSE_HOST}}``. Passwords can be left as
-   ``{{CLICKHOUSE_PASSWORD}}``, though be aware that if you are adding new
-   databases, you'll need to update ``SUPERSET_DB_PASSWORDS`` in the init scripts.
-   Here is the default connection string for reference::
+   tutor images build aspects-superset --no-cache
+   tutor local do import-assets
 
-   ``clickhousedb+connect://{{CLICKHOUSE_REPORT_URL}}``
-#. You will likely also run into issues where our SQL templates have been expanded into
-   their actual SQL. If you haven't changed the SQL of these queries (stored in
-   `tutoraspects/templates/openedx-assets/queries` you can just revert that change back
-   to their `include` values such as:
-   `sql: "{% include 'openedx-assets/queries/fact_enrollments_by_day.sql' %}"`
-#. The script will also warn about missing `_roles` in dashboards. Superset does not export
-   these, so you will need to manually add this key with the roles that are necessary to
-   view the dashboard. See the existing dashboards for how this is done.
-#. Re-build your ``aspects-superset`` image with `tutor images build aspects-superset --no-cache`
-#. Run the command `tutor aspects check_superset_assets` to confirm there are no
-   duplicate assets, which can happen when you rename an asset, and will cause import
-   to fail. The command will automatically delete the older file if it finds a duplicate.
-#. Check that everything imports correctly by running `tutor local do import-assets`
-   and confirming there are no errors.
-#. Double check that your database password did not get exported before committing!
-#. Commit and submit a PR with screenshots of your new chart or dashboards, along with an
-   explanation of what data question they answer.
+Autoscaling
+-----------
 
+Aspects supports autoscaling configurations for Ralph, Superset, and Superset Worker via the `Pod Autoscaling plugin <https://github.com/eduNEXT/tutor-contrib-pod-autoscaling>`_. Modify autoscaling settings as needed.
 
-Virtual datasets in Superset
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Contributing Charts and Dashboards
+===================================
 
-Superset supports creating virtual datasets, which are datasets defined using a SQL query instead of mapping directly to an underlying database object. Aspects leverages virtual datasets, along with `SQL templating <https://superset.apache.org/docs/installation/sql-templating/>`_, to make better use of table indexes.
+To contribute Superset assets:
 
-To make it easier for developers to manage virtual datasets, there is an extra step that can be done on the output of ``tutor aspects serialize``. The ``sql`` section of the dataset yaml can be moved to its own file in the `queries`_ directory and included in the yaml like so:
+1. Fork this repository and set up a local Tutor instance with Aspects installed.
+2. Export the desired assets using Superset’s “Export” feature.
```

Comment:
```suggestion
2. You should work on the non-localized versions of the Superset dashboards. Export the new or updated dashboard(s) using Superset’s “Export” feature. It is best to export the entire dashboard instead of just charts or datasets to ensure that all of the correct changes are captured.
```

Reply:
Changes resolved in commit : 
42f986d